### DISCUSSION TITLE
[Feature Request] Allow setting “min power” when using “dynamic power” option

### DISCUSSION THREAD

**Discussion by GizmoChicken** [ExpressLRS/ExpressLRS#1713] [Ideas]
Using the latest ELRS from the 3.x.x-maintenance branch, in conjunction with the latest ELRS lua script, I can set the “max power” when using the “dynamic power” option, which is great.  But I don't see a way to set the “min power” when using the “dynamic power” option. 

**Feature request:**  Please allow setting "**min power**" (in addition to "**max power**") when using the "**dynamic power**" option.

I often use the “dynamic power” option, typically with “max power” set to 500mW, just in case high power is needed.  And, honestly, the “dynamic power” option has generally worked great.  Even so, at least in some _unpredictable_ environments, allowing power to drop to as low as 10mW (the lowest power setting for my Jumper AION Nano 2400 TX) gives me pause.  In such _unpredictable_ environments, for peace of mind, I’d much prefer to set “min power” to 100mW and “max power” to 500mW.

Sure, I could turn off the “dynamic power” option, and set output to 100mW, 250mW, 500mW, or whatever my TX allows.  But I’d much prefer having the ability to set “min power” (in addition to “max power”) when using the “dynamic power” option.

P.S. Sorry if this has been previously requested; I searched for similar requests, but couldn’t find anything posted here.

**Comment by robustini** [comment]
I had already written this on Discord as a request a week ago, so since someone else has opened the discussion I join in.
